staking/programs/integrity-pool/src/context.rs

```diff
@@ -56,6 +56,28 @@ pub struct UpdateY<'info> {
     pub system_program: Program<'info, System>,
 }
 
+#[derive(Accounts)]
+pub struct UpdateDelegationFee<'info> {
+    #[account(mut)]
```

Comment:
You don't need a pyaer since you're not initializing any new accounts

Reply:
Fixed.